.circleci/config.yml

```diff
@@ -98,7 +98,6 @@ jobs:
             go version
             export GOMAXPROCS=2
             export PATH=$GOPATH/bin:$PATH
-            go test -p 2 -race -timeout 800s ./...
```

Comment:
I'd prefer if you comment this whole section with a TODO, not leave `test-prev-golang` but have it do nothing

Reply:
If I remove it I need to change also the settings of the whole project to not require it for the master branch ...

Reply:
Ah... Then please comment it out and leave a big visible `TODO`, and also open a `v0.28.0` issue to move k6 to Go 1.15 and re-enable these tests with 1.14 when we do so

Reply:
https://github.com/loadimpact/k6/issues/1589